articles/azure/azs-how-deploy-asdk.md

```diff
@@ -22,101 +22,209 @@ The Azure Stack Hub Development Kit (ASDK) is a single server instance of Azure
 ```
 
-The following details should be used:
+### [Physical Deployment](#tab/tabid-2)
 
-| Option        | Parameter                                                                          |
-|---------------|------------------------------------------------------------------------------------|
-| NTP           | 13.79.239.69                                                                       |
-| DNS Forwarder | 8.8.8.8                                                                            |
-| Drivers       | Browse to path of either the extracted Cisco drivers or the extracted VMware tools |
-| Computer Name | Anything but "azurestack", eg: "azurestackhost"                                    |
-| Static IP     | IP details assigned to the current interface                                       |
+No additional steps.
+
+***
+
+<br>
 
 Once step 3 is complete the box will have been rebooted from the vhdx downloaded above.
 
 ## Step 4 - Install ASDK
 
-Implement the following steps from the guide: [https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-run-powershell-script](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-run-powershell-script)
+Implement the following steps from the Microsoft documentation:
 
-- Deploy the development kit
+- [Deploy the development kit](https://docs.microsoft.com/en-us/azure-stack/asdk/asdk-install)
 
 The following details should be used:
 
-| Option        | Parameter                                                                                                                                                              |
-|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| NTP           | 13.79.239.69                                                                                                                                                           |
-| DNS Forwarder | 8.8.8.8                                                                                                                                                                |
-| Type          | Azure AAD, this should be either your own Azure AD account where you are the system admin or one you have setup for Azure Stack Hub. e.g. joebloggsukcloud.onmicrosoft.com |
-| Static IP     | Different IP than what you currently have - eg. 10.0.0.101 was my box 10.0.0.191 I set up - or just current IP + 1                                                     |
+| Option        | Parameter     |
+|---------------|---------------|
+| NTP           | 46.227.200.76 |
+| DNS Forwarder | 8.8.8.8       |
+| Type          | Azure AAD - this should be either your own Azure AD account where you are the system admin or one you have setup for Azure Stack Hub, for example `joebloggsukcloud.onmicrosoft.com` |
```

Comment:
I believe usually we use the same example as Microsoft: `contoso.onmicrosoft.com`.